src/test/java/racingcar/RacingGameTest.java

```diff
@@ -1,31 +1,85 @@
 package racingcar;
 
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.*;
 
 
 public class RacingGameTest {
 
-    private Car[] cars;
-    private MovableStrategy movableStrategy = new RandomMovableStrategy();
+    @MethodSource("provideValidArgument")
+    @ParameterizedTest
+    @DisplayName("RacingGame 객체 생성 테스트")
+    public void generateRacingGameTest(int round, String[] carNames) {
+        assertThat(new RacingGame(round, carNames)).isExactlyInstanceOf(RacingGame.class);
```

Comment:
클래스비교보단, 객체와 객체를 비교하는게 좋을 것 같습니다:)

Reply:
좀 모호해서 그러는데..구체적으로 객체와 객체 비교할 수 있는 예시하나만 주실 수 있을까요?  :)

Reply:
음...이 코멘트는 모호함을 넘어서 저의 실수였던 것 같네요...죄송합니다 ㅠ
객체 생성을 테스트하는 코드에서 객체와 비교하는 건 무리가 있겠네요...
코드를 다시 한번 보니, 철민님께서 구현하신데로 테스트 코드를 작성해도 괜찮을 것 같습니다!

개인적으로 첨언을 하자면,
```suggestion
        assertThatCode(() -> new RacingGame(round, carNames)).doesNotThrowAnyException();
```
저는 객체 생성을 테스트할 때, 이 방법을 사용합니다.
isExactlyInstanceOf()는 어떤 객체의 instance 타입을 정확히 확인해야 할때, (Ex. Exception의 종류나, 하위 클래스, 구현체 등을 확인할때) 사용하는 편이라서요.

다시 한번 혼란을 드려 죄송합니다 ㅠㅠ
이 부분도 개인적인 생각이니 참고하고 넘어가시면 될 것 같습니다!

Reply:
넵 꼼꼼히봐주셔서 고맙습니다